src/routes/(authenticated)/app/guilds/+page.svelte

```diff
@@ -1,8 +1,161 @@
 <script lang="ts">
+	import { ButtonSize } from '$lib/components/Base/Button/types';
+	
+	import JoyModal from '$lib/components/Advanced/Modal/JoyModal.svelte';
+	import { FontWeight, TextColor, TextSize } from '$lib/components/Base/Text/types'
+	import JoyButton from '$lib/components/Base/Button/JoyButton.svelte'
 	import { page } from '$app/stores'
 	import JoyContainer from '$lib/components/Base/Container/JoyContainer.svelte'
+	import JoyInput from '$lib/components/Base/Input/JoyInput.svelte'
+	import JoyText from '$lib/components/Base/Text/JoyText.svelte'
+	import { TextTag } from '$lib/components/Base/Text/types'
+	import {
+		Border,
+		BorderColor,
+		BorderRounded,
+		ContainerGap,
+		ContainerPadding,
+		Justify,
+	} from '$lib/types'
+	import { ButtonVariant } from '$lib/components/Base/Button'
+	import { AlignItems } from '$lib/types/AlignItems'
+	import { onMount } from 'svelte'
+	import { guilds, guildService } from '$lib/modules/guilds'
+	import { App } from '$lib/modules/app'
+	import { ToastVariant } from '$lib/components/Advanced/Toast/types'
+	import { type newGuildDTO } from '$lib/modules/guilds/types'
+	import { InputVariant } from '$lib/components/Base/Input/types'
+	import JoyTextArea from '$lib/components/Base/Input/JoyTextArea.svelte'
+	import type { PageData } from './$types'
+	import { superForm } from 'sveltekit-superforms'
+	const { loadGuilds } = guildService()
+	const { toast } = App
+
+	
+	export let data : PageData;
+
+	const {createGuild} = guildService()
+
+	
+	
+
+	onMount(async () => {
+		const err = await loadGuilds()
+
+		if (err) {
+			$toast.fire({ message: err.message, variant: ToastVariant.ERROR })
+		}
+	})
+
+
+	let isOpenModal = false;
+
+
+
+	const openModal = () => {
+		isOpenModal = !isOpenModal
+	}
+	
+
+
+	console.log($guilds)
+
+
+
+	const {form, enhance, errors, constraints } = superForm(data.guildValidSchema, {
+		dataType : 'json',
+		SPA : true,
+		resetForm : false,
+		validators : data.guildValidator,
+		onUpdate: async ({ form}) => {
+			if(!form.valid) return;
+
+		
+
+			
+			const err = await createGuild(form.data.name, form.data.description)
+		
+			if (err) {
+				return $toast.fire({ message: err.message, variant: ToastVariant.ERROR })
+			}
+
+		}
+	})
+
 </script>
 
-<JoyContainer class="bg-primary/50 w-full">
-	<h1>{$page.url.pathname}</h1>
+
+<JoyContainer
+	class="w-full h-screen overflow-y-auto"
+	col={true}
+	padding={ContainerPadding.XS}
+>
+
+<JoyContainer justify={Justify.BETWEEN} alignItems={AlignItems.CENTER} class="w-full">
+		<JoyInput placeholder="search..." />
+		<JoyContainer gap={ContainerGap.XXS}>
+			<JoyButton on:click={openModal} variant={ButtonVariant.PRIMARY}>Create</JoyButton>
+			<JoyButton variant={ButtonVariant.GHOST}>Refresh</JoyButton>
+		</JoyContainer>
+	</JoyContainer>
+
+	<div class="grid grid-cols-3 grid-flow-row gap-2 w-full">
+		{#each $guilds as guild (guild.id)}
```

Comment:
Hello @arielrecto , each guild item in the grid should have it's own component under `$lib/modules/guilds/components`